packages/css/src/tabs.css

```diff
@@ -5,80 +5,82 @@
   --dsc-tabs-content-padding: var(--ds-size-5);
   --dsc-tabs-content-color: var(--ds-color-neutral-text-default);
   --dsc-tabs-list-border-color: var(--ds-color-neutral-border-subtle);
-}
-
-.ds-tabs__panel {
-  padding: var(--dsc-tabs-content-padding);
-  color: var(--dsc-tabs-content-color);
-}
-
-.ds-tabs__tablist {
-  display: flex;
-  flex-direction: row;
-  border-bottom: var(--ds-border-width-default) solid var(--dsc-tabs-list-border-color);
-  position: relative;
-}
 
-.ds-tabs__tab {
-  align-items: center;
-  background: none;
-  border: 0;
-  box-sizing: border-box;
-  color: var(--dsc-tabs-tab-color);
-  cursor: pointer;
-  display: flex;
-  flex-direction: row;
-  font-family: inherit;
-  font-size: inherit;
-  gap: var(--ds-size-1);
-  justify-content: center;
-  line-height: var(--ds-line-height-sm);
-  margin: 0;
-  padding: var(--dsc-tabs-tab-padding);
-  position: relative;
-  text-align: center;
+  & u-tabpanel {
+    padding: var(--dsc-tabs-content-padding);
+    color: var(--dsc-tabs-content-color);
 
-  &:not([data-size]) {
-    font-size: inherit; /* Ensure inheriting font-size when <button> */
+    @composes ds-focus from './base.css';
   }
 
-  & :where(img, svg) {
-    flex-shrink: 0; /* Never shrink icon */
-    font-size: 1.25em; /* Auto scale icon based on font-size */
+  & u-tablist {
```

Comment:
Skulle vi hatt `& > u-tablist` og `& > u-tablist > u-tab` og `& > u-tabpanel` for å sikre at vi ikke styler nøstede elementer eller er det litt redundant tenker dere? :)

Reply:
Eg trur det er redundant, om du legger tabs i tabs så skal vel dei få samme stil uansett?

Reply:
Snakket med Mickey - vi legger på scope :)